File: 250429/DateTime to DateTime/datetime-to-datetime.cpp
#include <iostream>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(NULL);
    cout.tie(NULL);

    int day, hour, minute;
    cin >> day >> hour >> minute;

    int total_minutes = ((day * 24 * 60) + (hour * 60) + minute) - (11 * 24 * 60 + 11 * 60 + 11);
    if(total_minutes < 0) {
        cout << -1;
    }
    else {
        cout << total_minutes;
    }
    return 0;
}

File: 250429/DateTime to DateTime/datetime-to-datetime.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int start_day = 11;
        int start_hour = 11;
        int start_minute = 11;
        int day = sc.nextInt();
        int hour = sc.nextInt();
        int minute = sc.nextInt();
        boolean check = true;
        int total_minutes = 0;
        while(true) {
            if(start_day == day && start_hour == hour && start_minute == minute) {
                break;
            }
            if(day < start_day || (day == start_day && hour < start_hour) || (day == start_day && hour == start_hour && minute < start_minute)) {
                
                check = false;
                break;
            }
            total_minutes++;
            start_minute++;
            if(start_minute == 60) {
                start_minute = 0;
                start_hour++;
            }
            if(start_hour == 24) {
                start_hour = 0;
                start_day++;
            }
        }
        if(check) {
            System.out.println(total_minutes);
        } else {
            System.out.println(-1);
        }
        sc.close();
    }
}